README.md

```diff
@@ -4,26 +4,44 @@
 [![Packagist Version](https://img.shields.io/packagist/v/gheb/docusign-bundle.svg?style=flat-square)](https://packagist.org/packages/gheb/docusign-bundle)
 [![Software license](https://img.shields.io/github/license/gregoirehebert/docusign-bundle.svg?style=flat-square)](https://github.com/gregoirehebert/docusign-bundle/blob/master/LICENSE)
 
-This Bundle is used to create electronic signature with DocuSign.
-At the moment it only does handle implicit authentication with embedded signature.
-That means, that you need an account on DocuSign, and you'll be redirected to sign the document.
-
-DocuSign also offers the possibility to sign remotely, or to trigger a simple agreement click.
-But these options are not available yet.
-Feel free to contribute :)
+This Symfony Bundle is used to create electronic signature with DocuSign.
+An Electronic Signature ensure a person agreed with the document.
+
+1.  [Requirements](#requirements)
+1.  [Bundle Installation](#bundle-installation)
+    1.  [register the bundle](#register-the-bundle)
+    1.  [Configure the bundle](#configure-the-bundle)
+1.  [Docusign Configuration](#docusign-configuration)
+    1.  [Types of signatures](#types-of-signatures)
+    1.  [Format Restrictions:](#format-restrictions)
+    1.  [Add an integration key on docusign](#add-an-integration-key-on-docusign)
+1.  [Basic usage](#basic-usage)
+    1.  [Inside a twig template](#inside-a-twig-template)
+    1.  [Using document variations signature positionning](#using-document-variations-signature-positionning)
+1.  [Customization](#customization)
+    1.  [Configure a custom storage without league/flysystem-bundle](#configure-a-custom-storage-without-leagueflysystem-bundle)
+    1.  [Use league/flysystem-bundle](#use-league/flysystem-bundle)
+        1.  [Installation](#installation)
+        1.  [Configure the storage](#configure-the-storage)
+    1.  [Override configuration signature positions](#override-configuration-signature-positions)
+    1.  [Events](#events)
+        1. [Sign events](#sign-events)
+        1. [Webhook events](#webhook-events)
+    1.  [Using an external route as callback](#using-an-external-route-as-callback)
+1.  [Backward Compatibility promise](#backward-compatibility-promise)
```

Comment:
```suggestion
1.  [Backward compatibility promise](#backward-compatibility-promise)
```

Reply:
I want to keep the uppercase here as it refers to the BC accronym